### PR TITLE
Fixed llvm build in a container

### PR DESCRIPTION
**Pull request by npanpaliya**
## Checklist before submitting

- [ ] Did you read the [contributor guide](https://github.com/open-ce/open-ce/blob/main/CONTRIBUTING.md)?
- [ ] Did you update any affected [documentation](https://github.com/open-ce/open-ce/blob/main/doc/)?
- [ ] Did you write any tests to validate this change?

## Description

Fixes # (issue).

## Review process to land 

1. All tests and other checks must succeed.
2. At least one [maintainer](https://github.com/open-ce/open-ce/blob/main/MAINTAINERS.md) must review and approve.
3. If any  [maintainer](https://github.com/open-ce/open-ce/blob/main/MAINTAINERS.md) requests changes, they must be addressed.

### CODE REVIEW

**Review by pradghos — APPROVED** [review]
LGTM, Thanks !

**Review by cdeepali — APPROVED** [review]

File: envs/feedstock-patches/llvm/llvm-s390x.patch [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+
+diff --git a/recipe/meta.yaml b/recipe/meta.yaml
+index 99c275e..af7aa58 100644
+--- a/recipe/meta.yaml
++++ b/recipe/meta.yaml
+@@ -52,6 +52,7 @@ outputs:
+         - vs2017_{{ target_platform }}     # [win]
+         - clang_bootstrap_osx-64           # [osx]
+         - cmake
++        - make
```
Comment (ravigumm) [inline envs/feedstock-patches/llvm/llvm-s390x.patch:18]: Any idea why this is an issue only on s390x and not on other architectures ?

Comment (ravigumm) [reply envs/feedstock-patches/llvm/llvm-s390x.patch:18]: Got to know that this is not built on other architectures in open-ce

**Review by ravigumm — APPROVED** [review]
LGTM. Thanks

**Pull request merged**